scripts/runfabtests.sh

```diff
@@ -2,23 +2,35 @@
 
 trap cleanup_and_exit SIGINT
 
+#
+# Default behavior with no args will use sockets provider with loopback
+#
 declare BIN_PATH
-declare PROV
+declare PROV="sockets"
 declare TEST_TYPE="quick"
-declare SERVER
-declare CLIENT
+declare SERVER="127.0.0.1"
+declare SSH_SERVER
+declare CLIENT="127.0.0.1"
+declare SSH_CLIENT
 declare EXCLUDE
 declare GOOD_ADDR="192.168.10.1"
 declare -i VERBOSE=0
 
 # base ssh,  "short" and "long" timeout variants:
 declare -r bssh="ssh -n -o StrictHostKeyChecking=no -o ConnectTimeout=2 -o BatchMode=yes"
-declare -r sssh="timeout 60s ${bssh}"
-declare -r lssh="timeout 90s ${bssh}"
+declare TIMEOUT=`which timeout`
+if [ -z "$TIMEOUT" ]; then
```

Comment:
nit: simplify and suppress error message like: `if [ -z $(which timeout 2> /dev/null) ]; then`